pom.xml

```diff
@@ -28,6 +28,7 @@
 
   <properties>
     <spotbugs.failOnViolation>true</spotbugs.failOnViolation>
+    <jmh.version>1.21</jmh.version>
```

Comment:
From the maintainability reasons, we define versions of dependencies in the `kie parent ` https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml. This way we keep dependencies of all repositories in sync.

Would you please open a PR to the droolsjbpm-build-bootstrap, defining the jmh version there? At the same moment you can also remove it from kie-benchmarks.

Reply:
Good point, done.